### ISSUE TITLE
Focus on timeline on page load

### ISSUE THREAD

**Issue by jaranta** [nolanlawson/pinafore#99]
This is a tiny usability issue, but I run into this every time I open Pinafore.

## Expected

After page load, Page Down, Page Up or Space buttons scroll the timeline.

## Actual

Page Down, Page Up or Space buttons don't scroll the timeline. They start working after mouse clicking the area around the timeline.

**Comment by nolanlawson** [comment]
Good catch, looks like initially the focus is on the navigation instead of the timeline.

**Comment by mlcdf** [comment]
I can work on this one.

**Comment by mlcdf** [comment]
Not sure either. 

I guess the behaviour should be the same whether you are on the page displaying a timeline (local, federated, ) or not (settings, community, ...).

**Comment by nolanlawson** [comment]
fixed in #116 and #147
